tests/utils.py

```diff
@@ -167,7 +167,7 @@ class TestHomeServer(HomeServer):
 
 @defer.inlineCallbacks
 def setup_test_homeserver(
-    cleanup_func,
+    self,
```

Comment:
nit: could this be called test_object or something, since it's not a real instance method?